src/Console/Commands/DefaultCommand.php

```diff
@@ -88,8 +99,9 @@ private function renderIssue(OutputInterface $output, Issue $issue, string $curr
         renderUsing($output);
 
         $file = str_replace($currentDirectory, '.', $issue->file);
-        $lineInfo = ($issue->line !== 0) ? ":{$issue->line}" : '';
-        $suggestions = implode(', ', $issue->misspelling->suggestions);
+        $lineInfo = ($issue->line !== 0) ? " on line <strong>{$issue->line}</strong>" : '';
```

Comment:
I'm not sure why this line is different, I think it may have changed  during the process of me submitting the PR

Reply:
This commit apart from the main feature also added this change (from "on line ..." to ":{$line}").
https://github.com/peckphp/peck/commit/6ea4f711af2ea3e0e28444e46a45598ab05d0d31

Probably this was merged after you synced your fork. Therefore this changes back.
I personally don't mind either version!

Reply:
Hi @c0nst4ntin & thanks for the review. I agree... I think I had this commit on my fork for a few days, but I also had my 1st PR awaiting.  As they were both on the main branch, it wanted to include the the code from both. This is the reason I put all my PRs into separate branches.

I digress - I think it was in that time span that the code got crossed.

I thought I ignored that particular line when I commited the changes, but obviously not.

I'm not sure what to do now as I'm quite new to PRs. Can I change the code while it's in the PR phase, and revert line 91/102 to the latest version?

Reply:
I'm not sure if an actual revert will work (some platforms like bitbucket behave a bit different with this feature). But a regular commit (where you change everything back) will work, whilst the PR is open. 

If you commit after it is merged, the changes in your fork won't effect the upstream repository unless you open a new pull request.